src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -336,9 +336,18 @@
         Map<String, Object> caseData = caseDetails.getData();
 
         if (isApplicantSolicitorAgreeToReceiveEmails(caseDetails)) {
-            log.info("Sending email notification to Solicitor for 'Consent Order Available'");
-            notificationService.sendConsentOrderAvailableEmail(caseDetails);
+            log.info("Sending email notification to Applicant Solicitor for 'Consent Order Available'");
+            notificationService.sendConsentOrderAvailableEmailToApplicantSolicitor(caseDetails);
         }
+
+        if (featureToggleService.isRespondentJourneyEnabled()
+            && notificationService.shouldEmailRespondentSolicitor(caseData)) {
+            if (isConsentedApplication(caseDetails)) {
```

Comment:
@ravinderpal do we need to do this check? this endpoint only gets called from consented config. the call above also doesn't do this check

Reply:
if you remove this check you also won't need to update the case type ID in the test data

Reply:
Can the endpoint get called from ConsentedInContested? if yes, will this change affect it?

Reply:
@ravinderpal this endpoint isn't currently called by consent in contested endpoint. I think we'll cross that bridge when the require comes, I think it will need to call a different method to pull values from a different place anyhow